packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -1135,6 +1143,18 @@ export default defineComponent({
 				void this.$externalHooks().run('parameterInput.modeSwitch', telemetryPayload);
 			}
 		},
+		getEditorContent(editorType: string): string | null {
```

Comment:
do we still need this method? or should we inline it inside the computed prop?

Reply:
Nope, will move it back.